<ui version="4.0" >
 <class>SetPatternForm</class>
 <widget class="QWidget" name="SetPatternForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Установка образца</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Управление пуансоном</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="pbMoveDown" >
     <property name="text" >
      <string comment="Вниз" />
     </property>
     <property name="icon" >
      <iconset resource="../rc/resources.qrc" >
       <normaloff>:/Images/b_down.svg</normaloff>:/Images/b_down.svg</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pbMoveStop" >
     <property name="text" >
      <string comment="Стоп" />
     </property>
     <property name="icon" >
      <iconset resource="../rc/resources.qrc" >
       <normaloff>:/Images/b_stop.svg</normaloff>:/Images/b_stop.svg</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbMoveUp" >
     <property name="text" >
      <string comment="Вверх" />
     </property>
     <property name="icon" >
      <iconset resource="../rc/resources.qrc" >
       <normaloff>:/Images/b_up.svg</normaloff>:/Images/b_up.svg</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/resources.qrc" />
 </resources>
 <connections/>
</ui>
